tests/index.test.ts

```diff
@@ -587,7 +587,7 @@ describe('Utils', () => {
       it('Combines `testdouble` proxies from mocks generated via interfaces', async () => {
         const mock = td.object<ITestInterface>()
 
-        const result = await ResultAsync.combine([okAsync(mock)] as const)
+        const result = await ResultAsync.combine([okAsync(mock)])
```

Comment:
The `as const` here is no longer necessary, as typescript will now infer the correct type!

This situation comes up regularly in my code, and is the motivation for this PR. Loose example:
```ts
Result.combine([
  User.find(...),
  Contact.find(...),
]).map(([user, contact]) => {
  // Use user and contact
});
```